### PR TITLE
feat: rebuilding lambda package for al2023

### PR DESCRIPTION
**Pull request by mliner**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module rds-snapshotting-source
```
Comment (jaygridley) [inline go.mod]:
```suggestion
module github.com/lablabs/aws-rds-snapshotting-source
```

File: README.md [hunk 2]
```diff
@@ -61,6 +61,20 @@ No output.
 
 Feel free to create an issue in this repository if you have questions, suggestions or feature requests.
 
+### Building lambda package
+
+- use `asdf` to install necessary go version
```
Comment (jaygridley) [inline README.md:66]: We should add `.tool-versions` file to the repository with `golang` version

**Review by jaygridley — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (mliner) [reply go.mod]: updated

File: README.md (hunk 2 above)
Comment (mliner) [reply README.md:66]: added

**Review by jaygridley — APPROVED** [review]
LGTM

**Pull request merged**